### DISCUSSION TITLE
How to fix this error in 9.4.1

### DISCUSSION THREAD

**Discussion by KInGATiLLa** [vercel/next.js#13041] [Help]
![image](https://user-images.githubusercontent.com/17881842/82212117-9da6c680-9944-11ea-94d0-71b40e638fc0.png)

**This error doesn't appear and working in version 9.3.6**
but when i upgrade current next version to latest, then i got the error. 
**Is it my coding mistake?**  
Here is my code
[![Edit cocky-keldysh-e47iq](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/cocky-keldysh-e47iq?fontsize=14&hidenavigation=1&theme=dark)
These 2 files basemap.js map.js. BaseMap component is Map component's child

**Comment by Timer** [comment]
The CodeSandbox link provided is a CRA app and does not reproduce the issue.

Can you please share a Next.js one?

**Reply by KInGATiLLa** [reply]
ok sir, I can share all of my project files into codesandbox

**Reply by KInGATiLLa** [reply]
[![Edit dms-nextjs-app](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/dms-nextjs-app-r74bc?fontsize=14&hidenavigation=1&theme=dark)

But my project doesn't working on codesandbox. If you can check this link

**Comment by KInGATiLLa** [comment]
@Timer Hi 
Ok, I can share my code with codesandbox nextjs starter
